docs/agent/hostname-resolution.md

```diff
@@ -10,6 +14,12 @@ On upgrades from Agent v5 to Agent v6 (<6.6.0), this may make the Agent hostname
 
 Starting from the Agent v6.3 a configuration flag called `hostname_fqdn` has been introduced that allows the Agent v6 to have the same behavior as Agent v5. This flag is disabled by default on version 6.3 and enabled by default in version 6.6.
 
+## Windows
+
+On Agent v5, the ageent Windows reported the unqualified hostname by default.  To maintain backward compatibility, this behavior is preserved with Agent v6.  The new flag `hostname_fqdn` will remain disabled by default on Windows.
```

Comment:
typo: `agent`

Also:
* can we say `this default behavior is preserved with Agent v6`? (i.e. add `default` to make this clearer)
* the flag will remain disabled by default _on future v6 versions_, however we will likely want to change that in a future major version of the Agent.